core/src/main/scala/cats/data/Validated.scala

```diff
@@ -228,10 +227,25 @@ private[data] sealed abstract class ValidatedInstances extends ValidatedInstance
     def show(f: Validated[A,B]): String = f.show
   }
 
-  implicit def validatedBifunctor: Bifunctor[Validated] =
-    new Bifunctor[Validated] {
-      override def bimap[A, B, C, D](fab: Validated[A, B])(f: A => C, g: B => D): Validated[C, D] = fab.bimap(f, g)
-      override def leftMap[A, B, C](fab: Validated[A, B])(f: A => C): Validated[C, B] = fab.leftMap(f)
+  implicit def validatedBifunctor: Bitraverse[Validated] =
```

Comment:
It might be good to leave the `override def`s for `bimap` and `leftMap` in since they should be slightly more efficient than the versions derived from `Bitraverse`.